### DISCUSSION TITLE
HMIPW-DRD3 Kanal nicht erreichbar Servicemeldung obwohl inaktiv geschaltet.

### DISCUSSION THREAD

**Discussion by Xsez** [OpenCCU/OpenCCU#1272] [🇩🇪 Fragen&Antworten]
Ich habe einen (eigentlicht zwei) HMIPW-DRD3 schon lange am Bus hängen, dieser war bis vor kurzem jedoch nicht verdrahtet. Ich hab damals unter original CCU3 Firmware die Kanäle inaktiv geschaltet, damit die Servicemeldungen verschwinden. Vor einiger Zeit bin ich auf Raspberrymatic gewechselt und aktuell betreibe ich die CCU3 mit Raspberrymatic 3.57.5.20210424.

Die Tage habe ich den ersten Kanal verdrahtet und alle Kanäle aktiv geschaltet. Der erste Kanal konnte normal angesteuert werden und der zweite und dritte Kanal haben, wie zu erwarten, die Servicemeldung ausgegeben. Danach habe ich Kanal zwei und drei wieder inaktiv geschaltet, aber die Servicemeldung blieb weiterhin bestehen.

![Screenshot 2021-05-14 210404](https://user-images.githubusercontent.com/8041173/118382286-1d1cb380-b5f4-11eb-96ca-bea10a822433.png)
![Screenshot 2021-05-14 210431](https://user-images.githubusercontent.com/8041173/118382289-2279fe00-b5f4-11eb-84e8-80ede006fb68.png)

Nach einem Neustart der CCU3 tauchen die Servicemeldungen nach einigen Minuten wieder auf.

Ich habe auch noch einen zweiten HMIPW-DDR3 im System, der bisher auch ungenutzt ist. Ich habe bei diesem testweise die Kanäle aktiv geschaltet und wieder zurück auf inaktiv gesetzt. Nun habe ich drei weitere Servicemeldungen:

![Screenshot 2021-05-16 031231](https://user-images.githubusercontent.com/8041173/118382339-9ddbaf80-b5f4-11eb-8ad5-13066ba2eee4.png)

Den zweiten DDR3 habe ich auch schon einmal gelöscht, auf Werkseinstellungen gesetzt und neu angelernt. Nach anlernen sind alle drei Kanäle aktiv und die Servicemeldungen stehen an. Ich konfiguriere die Kanäle auf inkativ, aber die Servicemeldungen verschwinden nicht.

Leider hatte ich schon zu schnell ein Issue erzeugt:  https://github.com/jens-maus/RaspberryMatic/issues/1270
Bei Jens Maus scheint es jedoch zu gehen? Wahrscheinlich ist es überhaupt kein spezifisches Raspberrymatic Problem?

DDR3 mit Firmware 1.0.0, DRAP auf 2.2.24.

Den zweiten DDR3 kann ich ja nun erstmal unangelernt lassen. aber der erste wird aktiv eingesetzt...
Irgendwelche Ideen oder kennt das jemand?

**Comment by jens-maus** [comment]
In der Tat geht das bei mir problemlos im Wired Bus. Ich kann die Kanäle vom DRD3 usw. problemlos an/ausschalten und die Servicemeldungen verschwinden nach einer gewissen Zeit. Hast du mal probiert den gesamten Bus stromlos zu machen um zu sehen ob das vllt. die Servicemeldungen verschwinden lässt? Ggf. muss der DRD3 ja mitunter stromlos gemacht werden.
